conan/tools/apple/xcodedeps.py

```diff
@@ -8,22 +8,66 @@
 from conans.util.files import load, save
 from conan.tools.apple.apple import to_apple_arch
 
+GLOBAL_XCCONFIG_TEMPLATE = textwrap.dedent("""\
+    // Includes both the toolchain and the dependencies
+    // files if they exist
+
+    """)
+
+GLOBAL_XCCONFIG_FILENAME = "conan_config.xcconfig"
```

Comment:
This looks like a toolchain thing, not a dependencies thing, isn't it? Should it belong to the toolchain and not here'

Reply:
That file will aggregate both the info from the toolchain and the deps, it's used in both places